onnxruntime/core/framework/session_state.cc

```diff
@@ -214,6 +214,43 @@ void SessionState::CleanInitializedTensorsFromGraph() {
   graph_.CleanAllInitializedTensors();
 }
 
+Status SessionState::PrePackConstantTensors() {
+  // calculate the use count of each value
+  std::unordered_map<std::string, size_t> node_arg_use_count;
+  for (auto& node : GetGraphViewer().Nodes()) {
```

Comment:
Can this be const auto&? Explicitly call out const based on coding guidelines. "Qualify usages of 'auto' with 'const', '*', '&' and '&&' where applicable to more clearly express the intent".
Check other loops as well below.